tag/map_test.go

```diff
@@ -129,9 +129,7 @@ func TestNewMap(t *testing.T) {
 			name:    "from empty; invalid",
 			initial: nil,
 			mods: []Mutator{
-				Insert(k5, "v\x19"),
-				Upsert(k5, "v\x19"),
-				Update(k5, "v\x19"),
+				Insert(k5, strings.Repeat("x", 256)),
```

Comment:
Perhaps this could be a test of its own since the restriction being newly tested is `len(v) <= 255`

Reply:
added t.Run, so it should appear as it's own test

Reply:
Cool, thanks. LGTM!